jobs/config.json

```diff
@@ -28,6 +28,45 @@
       "sig-node"
     ]
   },
+  "ci-cri-containerd-e2e-gci-gce": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-cri-containerd-e2e-gce.env",
+      "--extract=ci/latest",
```

Comment:
I think after `--extract` cwd will become `/workspace/kubernetes`, before that it's inside the repo. Is this still correct?

Reply:
What?

Reply:
`--extract=ci/latest` will hit https://github.com/kubernetes/test-infra/blob/master/kubetest/main.go#L368-L372

Reply:
if you wanna merge and watch the actual behavior I'm also fine

Reply:
What about now?

Reply:
Is `extract=ci/latest` required for e2e test?

Reply:
ok :-)